content/hi/docs/reference/glossary/event.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Event
```

Comment:
Translate this please

Reply:
You can use `aka:` to also provide the original name.